torch/fx/experimental/fx2trt/converters/acc_ops_converters.py

```diff
@@ -241,16 +241,35 @@ def acc_ops_conv2d(network, target, args, kwargs, name):
     if has_dynamic_shape(input_val.shape):
         assert input_val.shape[1] != -1, "Channel dim can't be dynamic for convolution."
 
-    kernel = to_numpy(kwargs["weight"])
+    # for now we'll assume bias is constant Tensor or None,
+    # and bias being ITensor is not supported in TensorRT api
+    # right now
     bias = to_numpy(kwargs["bias"])
```

Comment:
Let's add a check in `to_numpy()` to fail loud if sees something that's not a `torch.Tensor`.